pkg/fileutil/files.go

```diff
@@ -90,23 +89,28 @@ func CopyDir(src string, dst string) error {
 		return err
 	}
 
-	entries, err := ioutil.ReadDir(src)
+	dirEntrySlice, err := os.ReadDir(src)
 	if err != nil {
 		return err
 	}
 
-	for _, entry := range entries {
-		srcPath := filepath.Join(src, entry.Name())
-		dstPath := filepath.Join(dst, entry.Name())
+	for _, de := range dirEntrySlice {
 
-		if entry.IsDir() {
+		srcPath := filepath.Join(src, de.Name())
+		dstPath := filepath.Join(dst, de.Name())
+
+		if de.IsDir() {
 			err = CopyDir(srcPath, dstPath)
 			if err != nil {
 				return err
 			}
 		} else {
+			deInfo, err := de.Info()
```

Comment:
Same issue here:  `DirEntry.Info()` has a two-value return so this isn't quite as clean as it used to be.